extensions/amp-form/0.1/amp-form.js

```diff
@@ -234,9 +234,15 @@ export class AmpForm {
         continue;
       }
 
-      if (input.type == 'checkbox' || input.type == 'radio') {
-        data[input.name] = input.checked ? input.value : '';
-      } else {
+
+      if (input.type != 'checkbox' && input.type != 'radio') {
+        data[input.name] = input.value;
+      } else if (input.type == 'checkbox' && input.checked) {
```

Comment:
Checkboxes and Radios shouldn't be added to params if none of the items in the group is not checked.